apps/taiga-lumbermill/src/dashboards/crypto/components/pools/pools.component.html

```diff
@@ -0,0 +1,115 @@
+@let data = tokens();
+@if (data) {
+    <div
+        tuiAppearance="whiteblock"
+        tuiCardLarge="normal"
+        [style.height]="'100%'"
+    >
+        <header tuiHeader>
+            <h2
+                tuiTitle
+                [style.text-align]="'center'"
+            >
+                Pools
+            </h2>
+        </header>
+        <tui-input
+            tuiTextfieldIconLeft="@tui.search"
+            [(ngModel)]="search"
+        >
+            Search
+        </tui-input>
+        <div class="list-pools">
+            <tui-scrollbar>
+                <cdk-virtual-scroll-viewport
+                    #viewport
+                    itemSize="64"
+                    tuiScrollable
+                    class="viewport tui-zero-scrollbar"
+                >
+                    <table
+                        tuiTable
+                        [columns]="columns"
+                    >
+                        <thead>
+                            <tr tuiThGroup>
+                                @for (column of columns; track $index) {
+                                    <th
+                                        *tuiHead="column"
+                                        tuiTh
+                                        [sticky]="true"
+                                        [style.top.px]="-viewport['_renderedContentOffset']"
+                                    >
+                                        {{ column }}
+                                    </th>
+                                }
+                            </tr>
+                        </thead>
+                        <tbody tuiTbody>
+                            <tr
+                                *cdkVirtualFor="let item of tableData() | tuiTableSort"
+                                tuiTr
+                            >
+                                @for (column of columns; track $index) {
+                                    <td
+                                        *tuiCell="column"
+                                        tuiTd
+                                        [style.padding-left]="'0.5rem'"
+                                    >
+                                        @if (column === 'Pair') {
```

Comment:
Unnecessary cycle with conditions. Each row contains three cells, so we can just make template without conditions